src/ucp/core/ucp_request.inl

```diff
@@ -762,6 +762,10 @@ ucp_request_complete_am_recv(ucp_request_t *req, ucs_status_t status)
                          req->user_data);
 }
 
+/*
+ * process data, complete receive if done
```

Comment:
document, assert(!offload)

Reply:
I checked that it can be used in HW unexpected RNDV flow. When HW RTS arrives unexpectedly, it is handled in very similar to the SW flow way.